src/NuGet.Clients/NuGet.VisualStudio.Implementation/Extensibility/VsPathContextProvider.cs

```diff
@@ -260,18 +261,18 @@ private IVsPathContext2 GetSolutionPathContext(string packagesFolderPath)
                 if (string.IsNullOrEmpty(projectNameFromMSBuildPath))
                 {
                     projectJsonPath = Path.Combine(msbuildProjectFile.DirectoryName,
-                        Common.ProjectJsonPathUtilities.ProjectConfigFileName);
+                        ProjectJsonPathUtilities.ProjectConfigFileName);
```

Comment:
why this file is being changed?

Reply:
namespace changes caused compiler errors between `NuGet.Common` and `NuGet.VisualStudio.Common`.